lib/JSON/Validator/OpenAPI.pm

```diff
@@ -83,7 +83,12 @@ sub validate_request {
         $value += 0;
       }
       elsif ($type eq 'boolean') {
-        $value = (!$value or $value eq 'false') ? Mojo::JSON->false : Mojo::JSON->true;
+        if (!$value or $value =~ /^(?:false|off)$/) {
+          $value = Mojo::JSON->false;
+        }
+        elsif ($value =~ /^(?:1|true|on)$/) {
```

Comment:
Where does “on” and “off" come from?

Reply:
Probably, this is one of the versions of true / false representation in some system. If so, it would be nice to add 't' / 'f' - in PostgreSQL it means true / false and I have to manually translate these values. But probably this should be moved to _validate_type_boolean () and change the conversion to boolean.